### DISCUSSION TITLE
v4 MySQL 5.7.39版本 连接不上

### DISCUSSION THREAD

**Discussion by Caisin** [rbatis/rbatis#297] [General]
报错信息 Backend(E("received corrupt message"))

**Comment by zhuxiujia** [comment]
> 报错信息 Backend(E("received corrupt message"))

This is because the driver uses SSL but the database does not have SSL enabled, so you try to add ```ssl-mode=disabled``` to the URL

**Comment by Caisin** [comment]
right, thank you!

**Comment by zhuxiujia** [comment]
The new version of the driver sets SSL-mode to disabled by default and simply performs Cargo Update
